### DISCUSSION TITLE
Completions not loaded for 3rd party language

### DISCUSSION THREAD

**Discussion by flexwie** [microsoft/vscode-discussions#1004] [Extension Development QnA] [answered]
I'm trying to write an extensions that provides extended support for variables in Azure Pipeline files.
The completions I defined for `azure-pipelines` are never displayed. If I change the language to `plaintext` the completions show up as expected.
Can the [official extension](https://github.com/microsoft/azure-pipelines-vscode) which provides the `azure-pipelines` language somehow prevent my completions?

Repo: https://github.com/flexwie/better-ado-extension

**Comment by r3m0t** [comment, marked as answer]
> Multiple providers can be registered for a language. In that case providers are sorted by their [score](https://code.visualstudio.com/api/references/vscode-api#languages.match) and groups of equal score are sequentially asked for completion items. The process stops when one or many providers of a group return a result. A failing provider (rejected promise or exception) will not fail the whole operation.

**Comment by flexwie** [comment]
So there is no way to display my completions along with the ones from the original package? I was somehow under the impression that they'd be merged.
